File: 01_Leetcode_FAANG/CHANDAN_8144/Concatenation_of_array.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        int s=2*nums.size();
        vector<int> ans;
        for(int i=0;i<s;i++)
        {
            int j=i%nums.size();
            ans.push_back(nums[j]);
        }
        return ans;
    }
};

File: 01_Leetcode_FAANG/Pratyush_Padihary/ConcatenationofArray.java
class Solution {
    public int[] getConcatenation(int[] nums) {
     int i,j,n= nums.length;
     int ans[]= new int[2*n];
        for(i=0;i<n;i++)
        {
            ans[i]=nums[i];
            ans[i+n]=nums[i];
        }
    return ans;
}}
